### ISSUE TITLE
Map Filter Panel - 3.3.2 Labels or Instructions - 1 empty form label is present on the Map Search filter dropdown

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9880]
As a user reliant on the screen reader, I expect all form controls to have accessible labels or instructions, telling me the name, purpose and type of form control receiving focus.

**Linked work item:**
#9493 

**Expected Result:**
All form controls should have labels or instructions to determine their purpose.

**Actual Result:**
An empty label on a form control means that a <label> element without any text in it is associated with the component. This means that users will not be told the name/purpose of the form control by the screen reader.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/6f83d205-da55-4ce1-9a41-1750103dfbb6)

**Comment by imogenroberts** [comment]
RETEST: The Map search filter drop-down now has an accessible form label.

**Evidence:**
![Map Filter - 3 3 2 Empty Form Label FIXED](https://github.com/archesproject/arches/assets/77161085/0bf1e146-45a7-48d2-9c2a-833ce80e1a1c)
